### PR TITLE
#52 - Create an Account Page

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds account Page

## Changes

- [x] Renamed to wallets.
- [x] Added fonts CDN.

## Screenshot

image

### CODE REVIEW

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -40,12 +41,15 @@
...
     "settings": "Settings",
     "notifications": "Notifications",
-    "inAppNotifications": "In app notifications",
-    "emailNotifications": "Email notifications",
-    "defaultSafeConfigs": "Default safe configs"
+    "inAppNotifications": "In app Notifications",
+    "emailNotifications": "Email Notifications",
+    "defaultSafeConfigs": "Default Safe configs",
+    "safesGuarding": "Safes you are Guarding",
+    "safesCreated": "Safes Created",
```
Comment (koshikraj) [inline src/i18n/locales/en.json]: Rename `safes` to `wallets` everywhere

File: src/screens/account/account.screen.tsx [hunk 2]
```diff
@@ -1,91 +1,95 @@
...
-        <Text variant='contentHeader' tx='accountPage.settings' left />
-        <Card>
-          <Text variant='contentHeader' tx='accountPage.notifications' left />
+        <StyledDiv>
+          <InfoContainer>
+            <IconContainer>
+              <Avatar name='guarding' size='medium' flat />
+            </IconContainer>
+            <HeadingContainer>
+              <Text variant='small' tx='accountPage.safesGuarding' />
```
Comment (koshikraj) [inline src/screens/account/account.screen.tsx]: We don't need the guarding section.

Comment (Koushith) [reply src/screens/account/account.screen.tsx]: removed in the latest commit.

**Comment by Koushith** [comment]
```
 <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap"
      rel="stylesheet"
    />
```

added this CDN link. This should fix font issues. if not, we can download the font and load it from local directory.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**